lib/custom_cops/spec_never_runs.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Getsafe
+  class SpecNeverRuns < RuboCop::Cop::Base
+    def on_new_investigation
+      path = processed_source.path
+      return unless path.include?('/spec/') && !path.end_with?('_spec.rb')
```

Comment:
I suggest to start using built-in `RSpec/FilePath` cop instead. 

But currently it produces 39 offences in insurance-service because we were not following guidelines for e.g. calculators spec.

Reply:
Hm, I'll take a look and see if we can ignore the guidelines part or adjust to it.

Reply:
Okay, what I'll do for now is:
- shelve this
- look into enabling the other cop

@Aerdayne since you're making a release soon, do you want to add in these other cops to your PR, or would you rather not? Up to you to enable or disable them tbh.

```rb
Lint/DuplicateMagicComment
Lint/UselessRescue
```

(I'd also recommend disabling `Style/TrailingCommaInArguments` if that's not problematic.)

If not, I'll wait and PR those later.

Reply:
I'll include `DuplicateMagicComment` and `UselessRescue`.
I'm against `TrailingCommaInArguments` though. Let's shelve it along with `MultilineMethodArgumentLineBreaks` for now.